### DISCUSSION TITLE
Add shortcuts for `[` and `]` to jump to start and end of data source

### DISCUSSION THREAD

**Discussion by defunctzombie** [foxglove/community#320] [Q&A]
Similar to the shortcuts for seek forward and backwards, the `[` and `]` shortcuts would jump to the start and end of the available source range.

**Comment by jtbandes** [comment]
Is there precedent for using <kbd>[</kbd> and <kbd>]</kbd> in other apps? What about the home/end keys which kind of inherently meant to jump to the beginning/end?
